api/resources_portal/models/material_request.py

```diff
@@ -12,6 +12,13 @@ class Meta:
 
     objects = models.Manager()
 
+    STATUS_CHOICES = (
+        ("PENDING", "PENDING"),
+        ("APPROVED", "APPROVED"),
+        ("REJECTED", "REJECTED"),
+        ("INVALID", "INVALID"),
```

Comment:
It just occurred to me that we should also have a `CANCELLED` state for if the requester cancels the request. This is different than `REJECTED` in that it was cancelled by the requester, rather than the sharer.